src/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en-US">
     <head>
         <meta charset="UTF-8">
-        <title>Red Hat Cloud Services Software | RBAC Service</title>
+        <title>RBAC</title>
```

Comment:
Please update this to `User Access Management` to match the navigation.  Thanks.

Reply:
Done